pkg/fqdn/cache.go

```diff
@@ -26,11 +26,6 @@ import (
 	"github.com/cilium/cilium/pkg/lock"
 )
 
-// DefaultDNSCache is a global, shared, DNS cache. It is the default cache used
-// by DNSPoller instances, unless initialized to use another.
-// Note: The DNSCache type returns all DNS information, regardless of source.
-var DefaultDNSCache = NewDNSCache()
```

Comment:
This might break some other things. I remember some monitor code referring to this directly. I don't think we're using it but hopefully CI compiles all important things.

Reply:
We already discussed this offline.